docs/guide/yaml/test/test-entities.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Blueprint Test Entities
+title_in_menu: Test Entities
+layout: website-normal
+---
+
+{% include fields.md %}
+
+## Structural Test Entities
+
+### TestCase
+The `TestCase` entity acts as a container for a list child entities which are started *sequentially*.
+{% highlight yaml %}
+{% readj example_yaml/testcase-entity.yaml %}
+{% endhighlight %}
+The ability to sequentially run entities is used to allow applications to fully deploy before attempting to start any test entities.
+
+If your application blueprint consists of multiple services then you will also require the `ParallelTestCase` entity.
```

Comment:
I was looking at it as the entity-under-test and the TestCase could execute concurrently. The first assertion could then be an assertion that within 10 minutes the serviceUp was true.

The sequential TestCase also allows a series of assertions, such as invoking an effector and then asserting that a sensor has the expected value (or that it is reachable using the http test, or whatever).